node/adapter/requirements.txt

```diff
@@ -15,6 +15,7 @@ pyOpenSSL==0.15.1
 pyasn1==0.1.7
 pyasn1-modules==0.0.5
 six==1.9.0
-urllib3==1.10.3
+urllib3>=1.7,<1.8
```

Comment:
Basically because I don't have an opinion either way. I can think of reasons for and against, but they don't move me so I just stick with the minimum modifications. But since you do have vocal opinions, I shall find the latest version in that range.